src/resources/LimitedRestorationToFull/BC.ts

```diff
@@ -24,7 +25,10 @@ export const BcRestorationResource: ResourceIF = {
       orgPersonLabel: 'Applicant',
       orgTypesLabel: 'Business or Corporation',
       subtitle: null // not used
-    }
+    },
+    nameRequestTypes: [
+      NrRequestActionCodes.RESTORE
```

Comment:
These are the NR Request Types that we allow for the subject filing (here, a LimitedRestorationToFull for a BC).